app/views/schools/transferring_participants/email.html.erb

```diff
@@ -7,13 +7,13 @@
 <div class="govuk-grid-row">
 	<div class="govuk-grid-column-two-thirds">
 
-		<span class="govuk-caption-xl"><%= @school.name %></span>
-    	<h1 class="govuk-heading-xl"><%= title %></h1>
-
-		<p class="govuk-body">You can enter their personal or school email address.</p>
-
 		<%= form_for @transferring_participant_form, url: { action: :email }, method: :put do |f| %>
-      		<%= f.govuk_text_field :email, label: { hidden: true }, width: "two-thirds" %>
+          <%= f.govuk_text_field :email, width: "two-thirds",
+              label: { text: title, tag: :h1, size: 'xl' },
+              caption: { text: @school.name, size: 'xl' } do %>
+
+              <p class="govuk-body">You can enter their personal or school email address.</p>
```

Comment:
This feels like it should be a hint. cc @etheya

Reply:
From [the design system](https://design-system.service.gov.uk/components/text-input/#hint-text):

> Use hint text for help that’s relevant to the majority of users, like how their information will be used, or where to find it.

This feels awfully hinty to me. Regardless, so long as it looks ok and is accessible I'm fine with it.